### ISSUE TITLE
Unable to build yowsup on windows

### ISSUE THREAD

**Issue by rahulserver** [tgalal/yowsup#891]
I followed instructions as given here: https://github.com/tgalal/yowsup#windows  

I installed python 3.4.3, then installed zlib and finally executed python setup.py install after downloading extracting the yowsup from hereenter link description here and changed directory to yowsup-master.

But I get this error:

```
c:/mingw/bin/../lib/gcc/mingw32/4.8.1/../../../../mingw32/bin/ld.exe: build\
temp.win-amd64-3.4\Release\curve25519module.o: bad reloc address 0x14 in section
 `.data'
    collect2.exe: error: ld returned 1 exit status
    error: command 'C:\\MinGW\\bin\\gcc.exe' failed with exit status 1
```
Then since above did not work, I tried using pip install yowsup2 but I get the same error. Could anyone get round this error? I am not into python, so not able to tell why is this error happening.

I am on windows 8.1, 64 bit.

**Comment by blackjack4494** [comment]
probably the same issue:
https://github.com/tgalal/yowsup/issues/674

i guess it is about python3

**Comment by rahulserver** [comment]
@blackjack4494 I get the same issue with python 2.6+ also. 
I just need to confirm if anyone has been able to successfully build it on windows.

**Comment by estar21** [comment]
@rahulserver I am having trouble installing on windows vista as well. Using Python 2.7 here. Seems to be an issue with the python-axolotl-curve25519 extension itself

**Comment by rahulserver** [comment]
@estar21 I assume that the owners of this repo should have tested it on some version of windows with some version of python before putting the instructions on home page. It would be great if they could put the exact version on which they could test it so that all of us who are struggling with making this work on windows could get it going quickly.

**Comment by estar21** [comment]
@rahulserver agree. I'm a noob. Would this qualify as a bug report? Which is the best/  right channel to flag it (developers vs github?) and are there official/ established places to flag these?
